clean.go

```diff
@@ -34,7 +34,7 @@ func clean(c *ec2macosinit.InitConfig) {
 		}
 		for _, d := range dir {
 			// Remove everything
-			err := os.RemoveAll(path.Join([]string{historyPath, d.Name()}...))
+			err := os.RemoveAll(filepath.Join([]string{historyPath, d.Name()}...))
```

Comment:
Hrm, not sure why this was dotted, can you fix this usage to avoid the slice literal?

Reply:
fixed ✅